rasa/utils/tensorflow/model_data.py

```diff
@@ -276,15 +312,16 @@ def get(
             sub_key: The optional sub key.
 
         Returns:
-            The requested data.
-        """
-        if sub_key is None and key in self.data:
-            return self.data[key]
-
-        if sub_key and key in self.data and sub_key in self.data[key]:
-            return self.data[key][sub_key]
-
-        return []
+            either the requested data (which can be a dictionary mapping
+            sub-keys to a list of `FeatureArray`s or just a list of
+            `FeatureArray`s in case there is no subkey), or
+            an empty list in case the requested data cannot be found
+        """
+        default = []
```

Comment:
yessss :)

Reply:
... but no we won't now - because I'd have to update every call to .get() now (or we drop None as possible default which we definitely won't do)

Reply:
You mean most `get` methods have `None` as a default and we'd have to write `default = []` which would be unusual? And if we wrote `default = None` then we'd have to modify all calls.

Reply:
> You mean most get methods have None as a default and we'd have to write default = [] which would be unusual?

yep, we'd probably have problems because that is an unexpected default value - and besides it's a mutable default which I just try to avoid everywhere

> And if we wrote default = None then we'd have to modify all calls.

yep